src/installLogsPrinter.d.ts

```diff
@@ -15,6 +16,12 @@ export type LogType = 'cons:log' |
   'cy:command' |
   'ctr:info';
 
+export type AllMessages = {
+  [specPath: string]: {
+    [testTitle: string]: [type: LogType, message: string, severity: Severity][]
+  }
+};
+  
```

Comment:
allows user to import this type for specifying arg to custom output processor i.e.
```
import { AllMessages } from 'cypress-terminal-report/src/installLogsPrinter'

export function customOutputProcessor(
  allMessages: AllMessages
) {

```